Makefile

```diff
@@ -0,0 +1,17 @@
+all: push
+
+# 0.0 shouldn't clobber any release builds
+TAG = 0.0
+PREFIX = kobolog/gorb
+
+binary: main.go
+	CGO_ENABLED=0 GOOS=linux godep go build -a -installsuffix cgo -ldflags '-w' -o docker/gorb ./main.go ./http.go
```

Comment:
Why `-installsufix cgo`? Also, I think it will build just fine without listing the source files in the end.